magento/magento2-page-builder
# Deprecate OpenPageBuilderFullScreen Action Group

### Description (*)
Adding deprecation message for `openPageBuilderFullScreen` action group.

### Fixed Issues
1. magento/magento2-page-builder#614: Deprecate OpenPageBuilderFullScreen Action Group

### Related Pull Requests
_N/A_


### Manual Testing Scenarios (*)
- [ ] **Confirm Deprecation Warning**
1. Create an MFTF test that utilises the `openPageBuilderFullScreen` action group
2. Run the test
3. Confirm deprecation warning appears

### Questions or Comments
Requires all tickets from the **Full Screen Only Page Builder Mode** to be merged (otherwise MFTF tests will fail). Currently awaiting magento/magento2-page-builder#512 and magento/magento2-page-builder#513.

### Checklist
